lib/compile.js

```diff
@@ -186,22 +214,22 @@ function compiler() {
 	dependencies = Object.keys( formattersAndParsers.map( functionName ).reduce(function( sum, i ) {
 		return extend( sum, DEPENDENCIES[ i ] );
 	}, {}));
-	properties.dependenciesAmd = JSON.stringify( dependencies.map(function( dependency ) {
+
+	properties.dependencies = dependencies.map(function( dependency ) {
 		return "globalize-runtime/" + dependency;
-	}));
-	properties.dependenciesCjs = dependencies.map(function( dependency ) {
-		return "require(\"globalize/dist/globalize-runtime/" + dependency + "\")";
-	}).join( ", " );
-	properties.dependenciesVars = formattersAndParsers
+	});
+	dependenciesVars = formattersAndParsers
 		.map( deduceDependenciesVars )
 		.reduce(function( sum, i ) {
 			return extend( sum, i );
 		}, {});
-	properties.dependenciesVars = Object.keys( properties.dependenciesVars )
+	dependenciesVars = Object.keys( dependenciesVars )
 		.map(function( dependency ) {
 			return "var " + dependency + " = Globalize._" + dependency + ";";
 		}).join( "\n" );
 
+	properties.compiled = dependenciesVars + "\n" + properties.compiled;
+
```

Comment:
It's still used for the template function, but I could change its signature to `function (code, dependencies)`.